src/ui/FacetRange/FacetRange.ts

```diff
@@ -117,14 +117,17 @@ export class FacetRange extends Facet implements IComponentBindings {
   }
 
   public getValueCaption(facetValue: FacetValue): string {
-    const lookupValueIsSpecified = !isUndefined(facetValue.lookupValue) && facetValue.lookupValue !== facetValue.value;
-    if (this.options.valueCaption || lookupValueIsSpecified) {
+    if (this.options.valueCaption || this.isLabelSpecifiedForValue(facetValue)) {
```

Comment:
Found a formatting issue with generated range 
related to https://coveord.atlassian.net/browse/JSUI-3006